### PR TITLE
Update docRegistry.txt to end of 2023

### PR DESCRIPTION
**Pull request by nedley**
I followed the instructions of docs/emoji/index.md and filled in the remaining lines through the end of 2023.

### CODE REVIEW

File: unicodetools/src/main/resources/org/unicode/tools/emoji/docRegistry.txt [hunk 1]
```diff
@@ -9092,4 +9092,175 @@ L2/23-113	GB 18030-2022 Amendment, Draft 1	China NB	2023-04-26
...
+L2/23-169	Proposed Update UTS #10, Unicode Collation algorithm (HTML version)	Ken Whistler, Markus Scherer	2023-07-10
+L2/23-170	Proposed Update UAX #14, Unicode Line Breaking Algorithm (HTML version)	Robin Leroy	2023-07-10
+L2/23-171	Proposed Update UTS #18, Unicode Regular Expressions (HTML version)	Mark Davis	2023-07-10
+L2/23-172	Proposed Update UAX #29, Unicode Text Segmentation (HTML version)	Christopher Chapman	2023-07-10
+L2/23-173	Proposed Update UAX #38, Unicode Han Database (HTML version)	Ken Lunde, Richard Cook	2023-07-10
+L2/23-174	Proposed Update UTS #39, Unicode Security Mechanisms (HTML version)	Mark Davis, Michel Suignard	2023-07-10
+L2/23-175	Proposed Update UAX #41, Common References for Unicode Standard Annexes (HTML version)	Ken Whistler, Rick McGowan	2023-07-10
+L2/23-176	Proposed Update UAX #44, Unicode Character Database (HTML version)	Ken Whistler	2023-07-10
+L2/23-177	Proposed Update UAX #45, U-Source Ideographs (HTML version)	Ken Lunde	2023-07-10
+L2/23-178	Proposed Update UTS #51, Unicode Emoji (HTML version)	Mark David, Ned Holbrook	2023-07-10
```
Comment (nedley) [inline unicodetools/src/main/resources/org/unicode/tools/emoji/docRegistry.txt:9157]: LOL "Mark David"?

File: unicodetools/src/main/resources/org/unicode/tools/emoji/docRegistry.txt [hunk 2]
```diff
@@ -9092,4 +9092,175 @@ L2/23-113	GB 18030-2022 Amendment, Draft 1	China NB	2023-04-26
 L2/23-114	Proposal to encode 603 urgently needed ideographs in UCS	Ken Lunde, USNB	2023-05-02
 L2/23-115	USNB Comments on Draft 2 of GB 18030-2020 Amendment 1 and recommendation for ISO/IEC 10646:2022 Amendment 2	Peter Constable	2023-05-03
 L2/23-116	Proposal to update Appendix F: Documentation of CJK Strokes in Unicode 16.0	Night Koo	2023-05-02
-L2/23-117	Handling of the Heh in Sindhi Text	Kamal Mansour	2023-05-05
\ No newline at end of file
+L2/23-117	Handling of the Heh in Sindhi Text	Kamal Mansour	2023-05-05
+L2/23-118	Proposal for superscript katakana letters for Ryukyuan languages	Gen Kojitani	2023-05-08
+L2/23-119	Core Spec note on click letters	Kirk Miller	2023-05-15
+L2/23-120	Dot behavior for U+06CC ARABIC LETTER FARSI YEH followed by U+0654 ARABIC HAMZA
+ABOVE	Lorna Evans	2023-05-15
```
Comment (macchiati) [inline unicodetools/src/main/resources/org/unicode/tools/emoji/docRegistry.txt:9099]: Malformed line (tool should have caught it. (hmmm). Change the \n to space.

File: unicodetools/src/main/resources/org/unicode/tools/emoji/docRegistry.txt [hunk 3]
```diff
@@ -9092,4 +9092,175 @@ L2/23-113	GB 18030-2022 Amendment, Draft 1	China NB	2023-04-26
...
+L2/23-218	Proposal to encode Geomantic Figures	George Pollard	2023-10-04
+L2/23-219	Proposal to add two Latin pharyngeal voiced fricative characters	Denis Moyogo Jacquerye	2023-10-04
+L2/23-220	Proposal to encode (disunify) two stroke characters in CJK Strokes block	Night Koo	2023-10-05
+L2/23-221	Proposal to modify the representative glyph of U+31D2 CJK STROKE P	Night Koo	2023-10-05
+L2/23-222	Proposal to Encode the Pabuchi Script	Biswajit Mandal	2023-10-05
+L2/23-223	Preliminary Proposal to Encode the Kurux Banna	Biswajit Mandal	2023-10-05
+L2/23-224	Next steps for ISO 12199	ISO/TC 37/SC 2 Scty	2023-10-05
+L2/23-225	Request to up date the G Source reference for U+320DD (IRG N2648)	Eiso Chan, H. W. Ho	2023-10-08
+L2/23-226	Proposal to add one character used for ancient geographical
+name related to Mazu belief and customs to UAX #45	Eiso Chan	2023-10-08
```
Comment (macchiati) [inline unicodetools/src/main/resources/org/unicode/tools/emoji/docRegistry.txt:9206]: Malformed line (tool should have caught it. (hmmm). Change the \n to space.

**Review by macchiati — CHANGES_REQUESTED** [review]
looks good except for those 2 lines.

**Review by macchiati — APPROVED** [review]

**Pull request merged**